lib/ansible/plugins/cliconf/aireos.py

```diff
@@ -85,3 +86,12 @@ def get(self, command, prompt=None, answer=None, sendonly=False, newline=True, c
     def get_capabilities(self):
         result = super(Cliconf, self).get_capabilities()
         return json.dumps(result)
+
+    @property
```

Comment:
Why are all of these `@property`s? They don't return anything, so I don't see what the use of that is.

Reply:
Fixed